app/Http/Controllers/ApiOrderController.php

```
@@ -0,0 +1,453 @@
+            'status' => 'success',
+            'balance' => ($user->balance),
+            'currency' => get_setting('currency_code'),
+        ];
+
+        return response()->json($result);
+    }
+
+    /**
+     * List of services
+     */
+    public function services($user, $request)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user'. (Unused Code Rules)

(UnusedFormalParameter)